src/mixer/audio.ts

```diff
@@ -144,6 +146,9 @@ class Player extends ((PlayerEmitter as unknown) as { new (): EventEmitter }) {
 
     (wavesurfer as any).backend.gainNode.disconnect();
     (wavesurfer as any).backend.gainNode.connect(engine.finalCompressor);
+    (wavesurfer as any).backend.gainNode.connect(
```

Comment:
I'd like to know if there are any perf impacts from per-channel analysis. I don't think there are, but my spidey sense is tingling.

Reply:
I know there's a disable toggle, but having numbers would make me extra happy :)

Reply:
From testing, it wasn't much slower, when I artificially slowed the CPU (in DevTools) by 4x, it was only a couple ms more (like 8ms vs 12ms avg). I've decreased the FFT size since that made the most perceivable perf difference at 4x slower/real time. Only seems to make about 15% extra of a core to add the 3 extra VU meters at full speed.

Reply:
4ms is 25% of a frame budget, but at 4x slowdown that's probably reasonable.